docs/src/main/asciidoc/_attributes.adoc

```diff
@@ -67,5 +67,15 @@
 // .
 :create-app-group-id: org.acme
 :create-cli-group-id: {create-app-group-id}
+// Attributes required for single-sourcing to downstream.
+:JDK-ver-other: 17
+:JDK-ver-latest: 21
+:JDK-ver-all: 17 or 21
```

Comment:
Maybe name them `jdk-version-...`. I'm really not a big fan of shortening things and let's try to keep things all lowercase.